tztrout/data.py

```diff
@@ -259,23 +256,27 @@ def _get_latest_tz_names(self, tz):
             dt -= datetime.timedelta(**self.TD_STEP)
         return tz_names
 
-    def _get_tz_identifiers_for_us_zipcode(self, zipcode):
-        """Get all the possible identifiers for a given US zip code."""
-        # Find the TZ identifier for a Zipcode given its latitude and longitude
-        # using the TimezoneFinder library
-        tz_id = tf.timezone_at(lng=zipcode.longitude, lat=zipcode.latitude)
-        return [tz_id]
-
     def generate_zip_to_tz_id_map(self):
         """Generate the map of US zipcodes to time zone identifiers. The
         method finds all the possible time zone identifiers for each zipcode
         using the latitude and longitude of each zipcode to search
         TimezoneFinder.
         """
+        from timezonefinder import TimezoneFinder
```

Comment:
It might be worth catching the `ImportError` and re-raising it with a better error message:
```python
try:
	from timezonefinder import TimezoneFinder
except ImportErrorL
	raise ImportError(
		'Dependency "timezonefinder" is missing. '
		'Install with "tz-trout[dev]" in order to generate timezone data'
	)
```